manifest.json

```
@@ -15,5 +15,42 @@
       "strict_max_version": "*",
       "primary_type": "adapter"
     }
+  },
+  "options": {
+    "schema": {
+      "type": "object",
+      "properties": {
+        "temperaturePrecision": {
+          "title": "Temperature precision",
+          "description": "The number of decimal places to show in temperature",
+          "type": "integer",
+          "enum": [ 0, 1, 2, 3 ],
+          "minimum":0,
+          "maximum":3
```

I don't think those are necessary if the enum property is set.

```suggestion
          "enum": [ 0, 1, 2, 3 ]
```